static/app/views/issueDetails/streamline/sidebar/seerDrawer.tsx

```diff
@@ -185,7 +185,7 @@ export function SeerDrawer({group, project, event}: SeerDrawerProps) {
       </SeerDrawerHeader>
       <SeerDrawerNavigator>
         <Flex align="center" gap={space(1)}>
-          <Header>{t('Autofix')}</Header>
+          <Header>{t('Seer')}</Header>
```

Comment:
Should it say Seer or Issue Fix or something else describing the feature inside this drawer?

Reply:
Either works. The flyover is opened when you click on a button inside a box called "Seer", so there's a "link" (visually) there. I think "Seer" is fine here.